### PR TITLE
[MODULAR] Ballistic Skyrat: Part 1 | Un-Goofsec's the hardlight guns, Defines 9x19mm Peacekeeper as its own caliber and various balance and qol changes

### PR DESCRIPTION
**Pull request by MassacreDemigirl**
## About The Pull Request
Un-Goofsec's the hardlight guns, Defines 9x19mm Peacekeeper as its own caliber and various balance and qol changes

## How This Contributes To The Skyrat Roleplay Experience
Makes some previously altered weapons back to their former power, removes exploiting lack of proper defining in order to load 9x25 into 9x19 and vice versa. CMG using the multi-sprite magazines also looks better. Overall provides the playerbase with a wider array of choice guns.

## Changelog

:cl: ErdinyoBarboza
add: Added new mechanics or gameplay changes
add: Vintorez, PCR-9, Pitbull and DTR-6 came back in ballistic flavor
del: Removed hardlight guns (modular_skyrat\modules\goofsec\code\projectiles.dm)
balance: Dozer now uses 9x25mm for its namesake.
balance: Zeta-6 now uses 10mm Magnum.
fix: 9x19mm now is properly defined and cant be loaded to 9x25mm magazines and vice versa.
imageadd: Added AP and Incendiary  magazines for "croon" and "g11" subtype at mags.dmi
imagedel: Removed  a few duplicate icons from mags.dmi
spellcheck: Gun descriptions clarify whether they use 9x19 or 9x25.
refactor: Added support for AP and Incendiary Magazines to Multi-Sprite Magazines
/:cl:

### CODE REVIEW

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Remove the hardlight messages from the OPFOR entires

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
Also for the love of god please resprite those hardlight guns, they're sour codersprites

File: code/__DEFINES/~skyrat_defines/projectiles.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #define CALIBER_473MM "473mm"
 #define CALIBER_6MM "6mm"
 #define CALIBER_10MMAUTO "10x25mm"
+#define CALIBER_9MMPEACE "9x19mm"
```
Comment (YakumoChen) [inline code/__DEFINES/~skyrat_defines/projectiles.dm:7]: Name them 9mm Peacekeeper / Peacekeeper 9mm / just "Peacekeeper" rounds, we're trying to reduce the number of gunnut caliber names.

Comment (MassacreDemigirl) [reply code/__DEFINES/~skyrat_defines/projectiles.dm:7]: Its used with defines so coders see CALIBER_9MMPEACE 
Until it gets a flavor change this is good enough

**Comment by MassacreDemigirl** [comment]
> Also for the love of god please resprite those hardlight guns, they're sour codersprites

Out of scope for this PR but they are no longer used. This reverts them back to their original sprites

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**